src/VisualStudio/Core/Test.Next/Remote/SerializationValidator.cs

```diff
@@ -94,11 +94,10 @@ public async Task<T> GetValueAsync<T>(Checksum checksum)
             var data = await GetRequiredAssetAsync(checksum).ConfigureAwait(false);
             Contract.ThrowIfNull(data.Value);
 
-            using var context = new SolutionReplicationContext();
```

Comment:
This is explained in https://github.com/dotnet/roslyn/pull/73117 (which this PR continues off from).